SPECS/mstflint/mstflint.spec

```
@@ -58,6 +58,9 @@ find %{buildroot} -type f -name '*.a' -delete
 %{_mandir}/man1/*
 
 %changelog
+* Thu Feb 23 2023 Elaheh Dehghani <edehghani@microsoft.com> - 4.21.0-3
+- Enabled 'adb-generic-tools' in build config (license: MIT)
```

I think no license info is required for this change.
```suggestion
- Enabled 'adb-generic-tools' in build config.
```